inc/packages/did/class-did.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * DID Interface.
+ *
+ * @package FAIR
+ */
+
+namespace FAIR\Packages\DID;
+
+interface DID {
+	/**
+	 * Get the DID type.
+	 *
+	 * One of plc, web.
+	 */
+	public function get_type() : string;
```

Comment:
@rmccue Are we changing this to `get_method()` and the corresponding constant in PLC/Web to `METHOD` for consistency with the spec? Or?

Reply:
Changing to `method` consistently makes sense to me; I picked `type` without really thinking about it, but sticking to the standard term is better.